src/utils/footer.tsx

```diff
@@ -2,177 +2,189 @@ import { FooterLinksType, PreLoginFooterLinksType } from "@pagopa/mui-italia";
-      title: t('footer.pre_login.followUs.title'),
-      socialLinks: [
-        {
-          icon: "linkedin",
-          title: t('footer.pre_login.followUs.linkedin.label'),
-          href: "https://it.linkedin.com/company/pagopa",
-          ariaLabel: t('footer.pre_login.followUs.linkedin.ariaLabel'),
-        },
-        {
-          title: t('footer.pre_login.followUs.twitter.label'),
-          icon: "twitter",
-          href: "https://twitter.com/pagopa",
-          ariaLabel: t('footer.pre_login.followUs.twitter.ariaLabel'),
-        },
-        {
-          icon: "instagram",
-          title: t('footer.pre_login.followUs.instagram.label'),
-          href: "https://www.instagram.com/pagopaspa/?hl=en",
-          ariaLabel: t('footer.pre_login.followUs.instagram.ariaLabel'),
-        },
-        {
-          icon: "medium",
-          title: t('footer.pre_login.followUs.medium.label'),
-          href: "https://medium.com/pagopa-spa",
-          ariaLabel: t('footer.pre_login.followUs.medium.ariaLabel'),
-        },
-      ],
-      links: [
-        {
-          label: t('footer.pre_login.accessibility.label'),
-          href:
-            windowURL &&
-            ACCESSIBILITY_PARTICULAR_LINK.LANDING_URL_PATTERN &&
-            windowURL.includes(ACCESSIBILITY_PARTICULAR_LINK.LANDING_URL_PATTERN)
-              ? ACCESSIBILITY_PARTICULAR_LINK.PARTICULAR_ACCESSIBILITY_URL
-              : "https://form.agid.gov.it/view/eca3487c-f3cb-40be-a590-212eafc70058/",
-          ariaLabel: t('footer.pre_login.accessibility.ariaLabel'),
-          linkType: "internal",
-        },
-      ],
-    },
-  });
+export const preLoginLinks: (
+  t: TranslationFunction,
+  lang: LangCode,
+  windowURL?: string
+) => PreLoginFooterLinksType = (t, lang, windowURL) => ({
+  // First column
+  aboutUs: {
+    title: undefined,
+    links: [
+      {
+        label: t("footer.pre_login.about_us.label"),
+        href: `${PAGOPA_HOME}/${lang}/societa/chi-siamo`,
+        ariaLabel: t("footer.pre_login.about_us.ariaLabel"),
+        linkType: "external",
+      },
+      {
+        label: t("footer.pre_login.pnrr.label"),
+        href: `${PAGOPA_HOME}/${lang}/opportunita/pnrr/progetti`,
+        ariaLabel: t("footer.pre_login.pnrr.ariaLabel"),
+        linkType: "external",
+      },
+      {
+        label: t("footer.pre_login.media.label"),
+        href: `${PAGOPA_HOME}/${lang}/media`,
+        ariaLabel: t("footer.pre_login.about_us.ariaLabel"),
+        linkType: "external",
+      },
+      {
+        label: t("footer.pre_login.work_with_us.label"),
+        href: `${PAGOPA_HOME}/${lang}/lavora-con-noi`,
+        ariaLabel: t("footer.pre_login.work_with_us.ariaLabel"),
+        linkType: "external",
+      },
+    ],
+  },
+  // Third column
+  resources: {
+    title: t("footer.pre_login.resources.label"),
+    links: [
+      {
+        label: t("footer.pre_login.resources.legal_notes.label"),
+        href: `/${lang}/note-legali`,
```

Comment:
use function into notification utility here and below

Reply:
Done!